mackerel-plugin-memcached/memcached_test.go

```diff
@@ -54,7 +54,9 @@ STAT conn_yields 1487476
 STAT bytes 621371972
 STAT curr_items 955652
 STAT total_items 2423543841
+STAT evicted_nonzero 10
```

Comment:
memcached's `stats` command doesn't output `evicted_nonzero` stats, according to memcached protocol document: https://github.com/memcached/memcached/blob/master/doc/protocol.txt

`evicted_nonzero` stats is only available as `evicted:{slab}:evicted_nonzero` from `stats items` command.